internal/cache/file/downloader/job_test.go

```diff
@@ -848,3 +850,23 @@ func (dt *downloaderTest) Test_validateCRC_ForTamperedFileWhenEnableCrcCheckIsFa
 	// Verify fileInfoCache update
 	dt.verifyFileInfoEntry(uint64(jobStatus.Offset))
 }
+
+func (dt *downloaderTest) Test_validateCRC_WheContextIsCancelled() {
+	objectName := "path/in/gcs/file2.txt"
+	objectSize := 10 * util.MiB
```

Comment:
use 20MiB for safer side and request 1MiB in Download function as it's possible that job completes by the time execution reaches to Assert block of this unit test.

Reply:
10MB works fine. If i request for 1MB there is chance that context is cancelled before entire 20MB is dowloaded, hence crc check is not invoked.